File: templates/base/fastapi/src/routes/health.py
import time

from fastapi import APIRouter

router = APIRouter()

_start_time = time.time()


@router.get("/health")
async def health():
    return {
        "status": "ok",
        "uptime": round(time.time() - _start_time, 2),
        "timestamp": int(time.time()),
    }

File: templates/base/express/src/routes/health.ts
import { Router } from "express";

const healthRouter = Router();

const startTime = Date.now();

healthRouter.get("/health", (_req, res) => {
  res.json({
    status: "ok",
    uptime: Math.floor((Date.now() - startTime) / 1000),
    timestamp: new Date().toISOString(),
  });
});

export { healthRouter };
